src/components/VirtualizedGrid/VirtualizedGrid.jsx

```diff
@@ -198,7 +205,7 @@ const VirtualizedGrid = forwardRef(
                 rowCount={calcRowCount}
                 onScroll={onScrollCB}
                 onItemsRendered={onItemsRenderedCB}
-                className="virtualized-grid-scrollable-container"
+                className={cx("virtualized-grid-scrollable-container")}
```

Comment:
why we did no added a css modules class name here?

Reply:
If modular class wasn't added it means it wasn't found in the stylesheet - this class exists just for overrides

Reply:
i think we need to add a css modules class also, so when we remove the plain selectors we will still have classes per each element.

Reply:
I agree with @3dyonic but approved for this not be a blocker.

Reply:
I believe it would be more clear and easier to checl if we leave it like this for now